plugins/angular.js

```diff
@@ -6,7 +6,8 @@
 'use strict';
 
 // See https://github.com/angular/angular.js/blob/v1.4.7/src/minErr.js
-var angularPattern = /^\[((?:[$a-zA-Z0-9]+:)?(?:[$a-zA-Z0-9]+))\] (.+?)\n(\S+)$/;
+var angularPattern = /^\[((?:[$a-zA-Z0-9]+:)?(?:[$a-zA-Z0-9]+))\] (.*?)\n?(\S+)$/;
```

Comment:
`?` is the only meaningful change here, isn't it? Changes in data callback are only refactoring